apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -581,6 +581,12 @@ impl BridgeQueryPlanner {
                     usage_reporting.stats_report_key = sig;
                 }
 
+                tracing::info!(
+                    "experimental_apollo_metrics_generation_mode: {:?}",
+                    self.configuration
+                        .experimental_apollo_metrics_generation_mode
+                );
```

Comment:
if your intent is to create a metric, that won't work like this. You should take a look [here](https://github.com/apollographql/router/blob/dev/apollo-router/src/configuration/metrics.rs#L176)

Reply:
Ah sorry, that was temporary code I should have deleted.